extensions/amp-stream-gallery/1.0/test/test-stream-gallery.js

```diff
@@ -15,7 +15,7 @@
  */
 
 import * as Preact from '../../../../src/preact';
-import {StreamGallery} from '../stream-gallery';
+import {StreamGallery} from '../component';
```

Comment:
Please rename this file as `test-component.js`

Reply:
Thanks, updated!